Library/Homebrew/test/rubocops/shell_cmd_spec.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require "rubocops/shell_cmd"
+
+describe RuboCop::Cop::FormulaAudit::ShellCmd do
+  subject(:cop) { described_class.new }
+
+  context "When auditing shell commands" do
+    it "system arguments should be separated" do
```

Comment:
I'm not sure what the difference between `contex` + `it` and `specify` is.

Reply:
Just whatever reads the best; they are pretty much equivalent.